### PR TITLE
adding few more functions from mod dp_tools

### PR DESCRIPTION
**Pull request by srikarm99**
Added few more esl outbound functions which were needed in my use case. Kindly review this and we can plan on adding more functions

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ repository = "https://github.com/KaranGauswami/freeswitch-esl.git"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-tokio = { version = "1", features = ["io-util", "net", "rt"] }
+tokio = { version = "1", features = ["full"] }
```
Comment (KaranGauswami) [inline Cargo.toml:14]: I think we dont need this full features here, everything else is fine. Thanks.

File: Cargo.toml [hunk 2]
```diff
@@ -22,4 +22,4 @@ uuid = { version = "1.2", features = ["v4"] }
 thiserror = "1.0"
 
 [dev-dependencies]
-tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
+tokio = { version = "1", features = ["full"] }
```
Comment (KaranGauswami) [inline Cargo.toml:25]: Same as above comment.

**Review by KaranGauswami — CHANGES_REQUESTED** [review]
Thanks for the PR, Everything else looks good.

**Comment by srikarm99** [comment]
Hi, I reverted the features, will also add some test cases and examples for the written functions

**Review by KaranGauswami — APPROVED** [review]

**Comment by KaranGauswami** [comment]
Thanks, Merged now.

**Pull request merged**